src/brevitas_examples/imagenet_classification/ptq/ptq_evaluate.py

```diff
@@ -246,8 +246,12 @@ def main():
         f"{'mb_' if args.graph_eq_merge_bias else ''}"
         f"{act_quant_calib_config}_"
         f"{args.weight_quant_calibration_type}_"
-        f"{'bnc' if args.calibrate_bn else ''}")
-    # @TODO: include added options in configurations here
+        f"{'bnc' if args.calibrate_bn else ''}"
+        f"{args.quant_format}"
```

Comment:
Can you put this after target_backend?
Also we need to add '_' after all the new options